### PR TITLE
docker: simplify examples

### PR DESCRIPTION
**Pull request by MasterOdin**
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](../blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](../blob/master/CONTRIBUTING.md#guidelines).

This is the first of PRs that is attached to the raised issue #2592, reverting the changes of #1505 to the more common real-world suggested usage, with the intent to create further docker command specific pages. 

I'm not sure how you'd want to handle the fact that it has 9 examples though. You could probably merge the two run examples, but not sure how to specify an optional `{{command}}` parameter.

### CODE REVIEW

File: pages/common/docker.md [hunk 1]
```diff
@@ -4,36 +4,36 @@
 
 - List currently running docker containers:
 
-`docker container ls`
+`docker ps`
```
Comment (sbrl) [inline pages/common/docker.md]: Was this meant to be `ls` instead of `ps`?

Comment (MasterOdin) [reply pages/common/docker.md]: Nope, that's the actual command. I assume it's one of the many reasons why they did the CLI restructure, adding in the newer sub-commands like `container`, `image`, etc.

**Comment by agnivade** [comment]
@MasterOdin - Could you remove one example from the page so that we are under our budget of 8 ?

File: pages/common/docker.md (hunk 1 above)
Comment (sbrl) [reply pages/common/docker.md]: I see. If the `container` commands are newer, why are we changing the page to the older commands? I'm confused.

Comment (pxgamer) [reply pages/common/docker.md]: @sbrl, see #2592. Looks like the short commands were going to be removed, but then most people tend to use the short commands. It looks like @MasterOdin was thinking of using additional pages for the long commands. So `docker-container`, etc.

Comment (sbrl) [reply pages/common/docker.md]: Ah, I see. This change is ok then :-)

Maybe once those pages are done we can revisit and add a `See also`.

**Comment by MasterOdin** [comment]
Removed a 9th example from the file, but I'm a bit conflicted. Would it be appropriate to add a `{{command}}` to the end of the `docker run` example and reword it to be "Start a container from an image with a custom name and run the command (leave off to use default Dockerfile command)" or something like that or just leave it as is?

File: pages/common/docker.md (hunk 1 above)
Comment (MasterOdin) [reply pages/common/docker.md]: Yeah, it's kind of a weird spot Docker is in. They added a bunch of sub-commands (container, image, etc.) to allow adding a lot more commands for those specific actions, but left the top level largely alone. As such, while all of the top level commands do have an equivalent action within a sub-command, most docs and things really just use these handful of key top level commands still.

**Comment by MasterOdin** [comment]
It's not mandatory and if not included, a default command will be executed by the developer of the Dockerfile. The most common usecase would be to not include the command though.

**Comment by sbrl** [comment]
Perhaps we should leave it off then. If it isn't commonly used, then I think it's better left to a more in-depth guide or tutorial :-)

File: pages/common/docker.md [hunk 2]
```diff
@@ -4,36 +4,32 @@
...
 `docker pull {{image}}`
 
-- Start a container from an image and get a shell inside of it:
-
-`docker container run -it {{image}} bash`
-
 - Run a command inside of an already running container:
 
-`docker container exec {{container_name}} {{command}}`
+`docker exec -it {{container_name}} {{command}}`
```
Comment (agnivade) [inline pages/common/docker.md]: I think it will be better if we change command to bash. That might preserve the information we already had in the page.

Comment (MasterOdin) [reply pages/common/docker.md]: Changed it to `sh` as that's more guaranteed to exist on a container than bash ([Alpine Linux](https://alpinelinux.org/) is a popular container image that doesn't have bash for example).

Comment (agnivade) [reply pages/common/docker.md]: Sure

File: pages/common/docker.md [hunk 3]
```diff
@@ -4,36 +4,32 @@
...
 
-- Start a container from an image and get a shell inside of it:
+- Open a shell inside of an already running container:
 
-`docker container run -it {{image}} bash`
-
-- Run a command inside of an already running container:
-
-`docker container exec {{container_name}} {{command}}`
+`docker exec -it {{container_name}} sh`
```
Comment (sbrl) [inline pages/common/docker.md]: Shouldn't `sh` be enclosed in `{{` `}}`?

Comment (MasterOdin) [reply pages/common/docker.md]: Done
